components/Storyblok/SbStoryPage/SbStoryFullView.tsx

```diff
@@ -92,16 +92,16 @@ export const SbStoryFullView = (props: SbStoryFullViewProps) => {
                      `}
               >
                 <Heading
-                  level={'h1'}
-                  weight={'semibold'}
-                  classes={`ood-story__title ood-has-tab-before su-before-bg-${props.blok.tabColor}`}
+                  as="h1"
+                  weight="semibold"
+                  className={`ood-story__title ood-has-tab-before su-before-bg-${props.blok.tabColor}`}
```

Comment:
The specific classes `ood-story__title` etc will be dealt with later when we get to the specific components